client/state/products-list/selectors/compute-full-and-monthly-prices-for-plan.js

```diff
@@ -57,7 +57,9 @@ function computePricesForWpComPlan( state, planObject ) {
 	const monthlyPlanObject = isMonthly
 		? planObject
 		: getPlan( getMonthlyPlanByYearly( planObject.getStoreSlug() ) );
-	const priceMonthly = getPlanRawPrice( state, monthlyPlanObject.getProductId(), true );
+	const priceMonthly = monthlyPlanObject
+		? getPlanRawPrice( state, monthlyPlanObject.getProductId(), true )
+		: 0;
```

Comment:
Note: Using 0 as the monthly price here is not a great solution for a **new** purchase -- it will lead to a weird display of the discounts (since it will try to compare the annual/biennial price to 0).  We could improve on that in the future.  But right now, this fixes the fatal error, and for the Blogger plan we only support renewals anyway, since it's not a product that we sell anymore.

Reply:
It would be nice if `priceMonthly` could safely be `undefined`. I recently converted the product variant code to TypeScript and when I typed that property I made it always a number, but it might be wise to make it able to be `undefined` there.

https://github.com/Automattic/wp-calypso/blob/a71676c4807d13c39e6a35a836cb51d4dae75f5a/client/my-sites/checkout/composite-checkout/hooks/product-variants.tsx#L40

It would be even nicer if this code had a return type, even if it was just jsdoc it would get picked up by the TS compiler. Anyway, these things can be done later.

Reply:
>Note: Using 0 as the monthly price here is not a great solution for a new purchase -- it will lead to a weird display of the discounts (since it will try to compare the annual/biennial price to 0). We could improve on that in the future.

Tracking that here: https://github.com/Automattic/wp-calypso/issues/50799

Reply:
It turns out that `priceMonthly` is unused. I took care of removing it in https://github.com/Automattic/wp-calypso/pull/50798/files